packages/manager/apps/pci-workflow/src/pages/new/utils/is-snapshot-consumption.ts

```diff
@@ -0,0 +1,4 @@
+export const isSnapshotConsumption = (
+  code: string | null | undefined,
+): boolean =>
+  typeof code === 'string' && code.startsWith('snapshot.consumption');
```

Comment:
how does code could be null or undefined ? it doesn't seem related to the snapshot consumption